roles/sap_ha_prepare_pacemaker/tasks/software_setup.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: "SAP Prepare Pacemaker - Check RHEL HA Repos"
+  ansible.builtin.shell: |
+    yum repolist
+  args:
+    executable: /bin/bash
+    warn: no
+  become: true
+  become_user: root
+  register: sap_ha_prepare_pacemaker_check_ha_repos
+  changed_when: false
+
+- name: "SAP Prepare Pacemaker - Subscription Manager - Enable RHEL HA Repos"
+  ansible.builtin.command: 'subscription-manager repos --enable="{{ item }}"'
+  loop: "{{ sap_ha_prepare_pacemaker_rhsm_repos }}"
+  when:
+    - item not in sap_ha_prepare_pacemaker_check_ha_repos.stdout
+
+- name: "SAP Prepare Pacemaker - Install Packages"
+  ansible.builtin.yum:
+    name: '{{ sap_ha_prepare_pacemaker_packages }}'
+    state: present
+
+- name: "SAP Prepare Pacemaker - Install Packages - Vendor Specific"
+  ansible.builtin.yum:
+    name: "{{ vars['sap_ha_prepare_pacemaker_packages_' + sap_ha_prepare_pacemaker_type] }}"
+    state: present
+  when:
+    - sap_ha_prepare_pacemaker_type is defined
+    - vars['sap_ha_prepare_pacemaker_packages_' + sap_ha_prepare_pacemaker_type] is defined
+
+## AZ specific repo setup
```

Comment:
Please append additional comment line, explaining why this code block is commented out? This will be necessary for an end user looking at the file in future

Reply:
Removing this block to add it in an improved version lateron which will support platforms more flexibly.

Reply:
Tracked under GH Issue #164